### PR TITLE
crimson/seastore: add --seastore-devs in vstart.sh

### PR DESCRIPTION
**Pull request by liu-chunmei**
to support /dev/xxx as seastore device

Signed-off-by: chunmei-liu <chunmei.liu@intel.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by liu-chunmei** [comment]
@athanatos  please review add --seastore-devs in vstart.sh

File: src/vstart.sh [hunk 1]
```diff
@@ -899,12 +911,15 @@ EOF
             if command -v btrfs > /dev/null; then
                 for f in $CEPH_DEV_DIR/osd$osd/*; do btrfs sub delete $f &> /dev/null || true; done
             fi
+            mkdir -p $CEPH_DEV_DIR/osd$osd
```
Comment (athanatos) [inline src/vstart.sh:914]: This would break the filestore_path and kstore_path portions below.

Comment (liu-chunmei) [reply src/vstart.sh]: @athanatos updated.

**Comment by athanatos** [comment]
LGTM, @tchaikov Does this look ok to you?

**Review by athanatos — APPROVED** [review]

**Review by tchaikov — APPROVED** [review]
will create a follow-up change to consolidate the repeating code.

**Comment by tchaikov** [comment]
#41542

**Pull request merged**